src/init.cpp

```diff
@@ -114,6 +114,10 @@ static StartupShutdownHandler g_oStartupShutdownHandler;
   
 /*****************************************************************************/
 
+#ifdef _MSC_VER
+// needed to get it to compile on msvc
```

Comment:
Hmm... swh seems to provide this for MINGW builds as well.  Should we copy? https://github.com/search?q=repo%3Aswh%2Fladspa%20__declspec&type=code

Reply:
For MinGW, it's optional because under the hood, it's gcc and __attribute__(constructor) does a similar thing there. Anyways nothing wrong with that approach either. Need someone with better knowledge to comment on that.

Reply:
Thanks for the explanation, merging.